Documentation/installation/servicemesh/ingress.rst

```diff
@@ -24,6 +24,15 @@ an existing K8s cluster with Cilium installed.
 
 .. include:: installation.rst
 
+Supported Ingress Annotations
+#############################
+
+- ``io.cilium/tcp-keep-alive``
```

Comment:
what do you think about having a table with 3 columns (name, description, and default value if applicable) ?